scripts/datacube.py

```diff
@@ -433,36 +345,37 @@ def process(
     Returns:
     - xr.Dataset: Merged xarray Dataset containing processed data.
     """
-
-    date, YEAR, MONTH, DAY, CLOUD = get_conditions(year1, year2, cloud_cover_percentage)
-    week = get_week(YEAR, MONTH, DAY)
-
-    catalog, s2_items, BBOX, epsg = search_sentinel2(
-        week, aoi, cloud_cover_percentage, nodata_pixel_percentage
+    year = random.randint(2017, 2023)
```

Comment:
Don't think we should hard code years into the function. Those should be placed back as args